Session/Meta/Session-Info.plist

```diff
@@ -66,17 +66,17 @@
 	<dict>
 		<key>NSExceptionDomains</key>
 		<dict>
-			<key>public.loki.foundation</key>
```

Comment:
We may still need `public.loki.foundation` in the exception domains to use testnet.

Reply:
Yea we will, but since testnet uses HTTP we actually need to add `NSExceptionAllowsInsecureHTTPLoads` in order for it to work properly anyway - wasn't sure if we wanted to leave it in since we need to modify it anyway when swapping to testnet?

Reply:
Yea, that makes sense. We need to modify the settings to use testnet anyway, no need to add `public.loki.foundation` back right now.